pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -679,18 +679,26 @@ async def publish_client(self, base_to_mirror_dir: str, pullspec, release_name,
             image_stat, oc_mirror_pullspec = get_release_image_pullspec(pullspec, "oc-mirror")
             if image_stat == 0:  # image exist
                 # extract image to workdir, if failed it will raise error in function
-                extract_release_binary(oc_mirror_pullspec, [f"--path=/usr/bin/oc-mirror:{client_mirror_dir}"])
-                # archive file
-                with tarfile.open(f"{client_mirror_dir}/oc-mirror.tar.gz", "w:gz") as tar:
-                    tar.add(f"{client_mirror_dir}/oc-mirror", arcname="oc-mirror")
-                # calc shasum
-                with open(f"{client_mirror_dir}/oc-mirror.tar.gz", 'rb') as f:
-                    shasum = hashlib.sha256(f.read()).hexdigest()
-                # write shasum to sha256sum.txt
-                with open(f"{client_mirror_dir}/sha256sum.txt", 'a') as f:
-                    f.write(f"{shasum}  oc-mirror.tar.gz\n")
-                # remove oc-mirror
-                os.remove(f"{client_mirror_dir}/oc-mirror")
+                multi_rhel_path = [f"--path=/usr/bin/oc-mirror.{rhel}:{client_mirror_dir}" for rhel in ["rhel8", "rhel9"]]
+                ret, _ = extract_release_binary(oc_mirror_pullspec, multi_rhel_path)
```

Comment:
only 4.15&4.16 has multi oc-mirror, and error will be raised and stopped in the function

I suggest use regex to extract, --path=/usr/bin/oc-mirror*:{client_mirror_dir}  and put all extracted file in oc-mirror.tar.gz